.github/workflows/APIs_tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Test dotnet APIs
+
+on:
+  workflow_call:
+    inputs:
+      path_to_project:
+        required: true
+        type: string
+
+jobs:
+  test_apis:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.path_to_project }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '7.0.x'
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build APIs
+        run: dotnet build --no-restore Backend.sln
```

Comment:
Jak budujemy solucję, to budujemy chyba tylko projekt startowy? Bo my w ramach jednej solucji mamy kilka API

Reply:
Nie, budują się wszystkie projekty dołączone do danej solucji